synfig-studio/src/gui/widgets/widget_timeslider.h

```diff
@@ -56,9 +56,18 @@ class Widget_Timeslider: public Gtk::DrawingArea
 
 	Cairo::RefPtr<Cairo::SurfacePattern> play_bounds_pattern;
 
+	Glib::RefPtr<Gdk::Cursor> bounds_cursor = Gdk::Cursor::create(get_display(), "ew-resize");
+	Glib::RefPtr<Gdk::Cursor> default_cursor = Gdk::Cursor::create(get_display(), "default");
```

Comment:
@ice0 what do you think about this initialization here in header?

Reply:
I think it's ok, but it might be better to move it to a .cpp file for this reason:

> However, if we decide to change the default value, it will force recompilation of any file that #includes Widget_TimeSlider.h